test/e2e/kubectl/kubectl.go

```diff
@@ -92,19 +83,34 @@ const (
 	pausePodSelector         = "name=pause"
 	pausePodName             = "pause"
 	runJobTimeout            = 5 * time.Minute
-	busyboxImage             = "gcr.io/google_containers/busybox:1.24"
-	nginxImage               = "gcr.io/google_containers/nginx-slim:0.7"
-	newNginxImage            = "gcr.io/google_containers/nginx-slim:0.8"
 	kubeCtlManifestPath      = "test/e2e/testing-manifests/kubectl"
-	redisControllerFilename  = "redis-master-controller.json"
+	redisControllerFilename  = "redis-master-controller.json.in"
 	redisServiceFilename     = "redis-master-service.json"
-	nginxDeployment1Filename = "nginx-deployment1.yaml"
-	nginxDeployment2Filename = "nginx-deployment2.yaml"
-	nginxDeployment3Filename = "nginx-deployment3.yaml"
-	redisImage               = "gcr.io/k8s-testimages/redis:e2e"
+	nginxDeployment1Filename = "nginx-deployment1.yaml.in"
+	nginxDeployment2Filename = "nginx-deployment2.yaml.in"
+	nginxDeployment3Filename = "nginx-deployment3.yaml.in"
 )
 
 var (
+	nautilusImage = imagemanifest.GetnautilusImage()
+	kittenImage   = imagemanifest.GetkittenImage()
+	redisImage    = imagemanifest.GetredisImage()
+	nginxImage    = imagemanifest.GetnginxSlimImage()
+	busyboxImage  = imagemanifest.GetBusyboxImage()
+)
+
+var funcMap = template.FuncMap{
```

Comment:
Can we use a normal `map[string]string` instead so we can use the more common `{{ .Field }}` format?